vosekast_control/MQTT.py

```diff
@@ -54,11 +57,13 @@ def connected(self):
         return self.client.is_connected
 
     def publish(self, topic, message):
-        self.client.publish(topic, message, qos=0)
-        # print("Published: \"" + message + "\" to topic: \"" + topic + "\"")
+        if self.connection_test():
+            self.client.publish(topic, message, qos=0)
+            # print("Published: \"" + message + "\" to topic: \"" + topic + "\"")
 
     def publish_message(self, message_object):
-        self.publish(message_object.topic, message_object.get_json())
+        if self.connection_test():
```

Comment:
Is it necessary to check the connection twice?

Reply:
will be fixed soon, thank you.